test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/TemplateSandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Template test sandbox
```

Comment:
The readme isn't very discoverable, unfortunately. I would suggest creating a new doc in https://github.com/dotnet/extensions/tree/main/docs (say, call it "testing.md"), adding a paragraph on how to test the templates and then link to this doc.

Reply:
Good idea!

Reply:
Hm, `building.md` already has general testing instructions, and having a `testing.md` might make those original instructions less discoverable. I could move all testing-related stuff to `testing.md`, but then the testing info for this project could get buried in other, unrelated details.

I've instead added a new README at the root of the project folder. I think that should be reasonably discoverable. Does that sound fine?

Reply:
I'd add the link to the new docs/testing.md to the main readme, so that contributors have a clear(er) direction. 

But I leave it with you to decide. :)